controllers/telemetry/tracepipeline_controller_test.go

```diff
@@ -221,7 +221,7 @@ var _ = Describe("Deploying a TracePipeline", Ordered, func() {
 		}, timeout, interval).Should(Succeed())
 	})
 
-	It("updates Trace Collector Secret when referenced secret changes", func() {
+	It("updates Trace Gateway Secret when referenced secret changes", func() {
```

Comment:
```suggestion
	It("updates trace gateway Secret when referenced Secret changes", func() {
```

Gateway = lower case according to https://opentelemetry.io/docs/collector/deployment/gateway/

Is the 2nd occurrence of "secret" not a Kubernetes resource (which are upper case acc to [our guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalization)) ?